networking/synthetic/stac-prepare-nodes.yaml

```diff
@@ -2,7 +2,39 @@
 - hosts: stac_nodes
   remote_user: root
   tasks:
-    
+
+  - name: install epel 7
+    yum:
+      name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+      state: present
```

Comment:
we call that ansible script from the prepare-nodes.py 

actually it is not deafult, I've got some gear without any epel repos.